### PR TITLE
include 'shell command' as a possible step task

### LINKED ISSUE #2210: 'Shell command' as alternative type of task for a step is missing
Hello! In the very useful [Introduction to GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/introduction-to-github-actions) page, specifically in the [Understanding the workflow file](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/introduction-to-github-actions#understanding-the-workflow-file) section, there's an explanation of what the `steps:` section is, which includes the following [sentence](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/introduction-to-github-actions#:~:text=Each%20line%20nested%20under%20this%20section%20is%20a%20separate%20action.): 

"_Each line nested under this section is a separate action._"

In a couple of places elsewhere on the same page, it is stated that "a step can be either an action or a shell command" - which is an important point. 

Therefore I suggest we add this fact to the sentence above, so it becomes:

"_Each line nested under this section is a separate action or shell command._"

I feel that the distinction and relationship between actions and shell commands deserves to be highlighted where appropriate, including here. Thank you!

### PR DESCRIPTION
**Pull request by qmacro**
### Why:

This page helped me greatly to understand all the component parts in GitHub Actions. One thing that is now clear to me is that there are two types of "task" that a step can actually be - either an action, or a shell command. This is explained in a couple of places in this page, but is notably missing in this specific "Understanding the workflow file" section. 

I've raised issue #2210 for this, and here's the PR to address it. Thanks. 

### What's being changed:

- Old: "Each line nested under this section is a separate action."
- New: "Each line nested under this section is a separate action or shell command."

### Check off the following:
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by Legionsixxx9 — COMMENTED** [review]
[]()

**Comment by janiceilene** [comment]
@qmacro Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Comment by qmacro** [comment]
Adding a comment to give this PR chance to survive the holiday period :)

**Comment by qmacro** [comment]
...

**Review by FAISAILWE23 — APPROVED** [review]

**Review by FAISAILWE23 — APPROVED** [review]

File: content/actions/learn-github-actions/introduction-to-github-actions.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ To help you understand how YAML syntax is used to create a workflow file, this s
   ```
 </td>
 <td>
-  Groups together all the steps that run in the <code>check-bats-version</code> job. Each line nested under this section is a separate action.
+  Groups together all the steps that run in the <code>check-bats-version</code> job. Each line nested under this section is a separate action or shell command.
```
Comment (lucascosti) [inline content/actions/learn-github-actions/introduction-to-github-actions.md:149]: 👋 Thanks a lot for this suggestion, @qmacro! Your change is great, but I'll make another teeny change to this sentence too. 

Although the workflow being described has only one line per step, generally each step isn't always only line (and mostly isn't). So, I've changed `line`->`item` to be a bit more correct too. 🙂 

```suggestion
  Groups together all the steps that run in the <code>check-bats-version</code> job. Each item nested under this section is a separate action or shell command.
```

**Review by lucascosti — APPROVED** [review]
Stage looks good! I'll merge this when the tests pass.

Thanks again, @qmacro! 🙇‍♂️

**Pull request merged**